app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -5,21 +5,29 @@
 module V0
   class BenefitsClaimsController < ApplicationController
     def index
-      claims = service.get_claims
+      claims = service.get_claims(settings.access_token.client_id, settings.access_token.rsa_key)
 
       render json: claims
     end
 
     def show
-      claim = service.get_claim(params[:id])
+      claim = service.get_claim(params[:id], settings.access_token.client_id, settings.access_token.rsa_key)
 
       render json: claim
     end
 
     private
 
     def service
-      @service ||= BenefitsClaims::Service.new(@current_user.icn)
+      @service ||= BenefitsClaims::Service.new(@current_user.icn, @current_user.ssn)
+    end
+
+    def settings
+      @settings ||= Settings.lighthouse.benefits_claims
+    end
+
+    def settings
```

Comment:
oopsie :hand_over_mouth:

Reply:
Thanks for catching!

Reply:
Get rid of this entirely? What's super weird is I don't think I made these changes (or any in this file). My gitlens says Eddie and Jerek

What I think I _did_ do is botch the merge with a mass "accept from source" haha